README.md

```diff
@@ -1,6 +1,64 @@
-### MIND MAPP
+# MIND MAPP
 
-Mind Mapp: An interactive mind mapping tool for programmers
+```
+.        :   ::::::.    :::.:::::::-.  
+;;,.    ;;;  ;;;`;;;;,  `;;; ;;,   `';,
+[[[[, ,[[[[, [[[  [[[[[. '[[ `[[     [[
+$$$$$$$$"$$$ $$$  $$$ "Y$c$$  $$,    $$
+888 Y88" 888o888  888    Y88  888_,o8P'
+MMM  M'  "MMMMMM  MMM     YM  MMMMP"`
+
+.        :    :::.  ::::::::::.::::::::::.
+;;,.    ;;;   ;;`;;  `;;;```.;;;`;;;```.;;;
+[[[[, ,[[[[, ,[[ '[[, `]]nnn]]'  `]]nnn]]'
+$$$$$$$$"$$$c$$$cc$$$c $$$""      $$$""    
+888 Y88" 888o888   888,888o       888o     
+MMM  M'  "MMMYMM   ""` YMMMb      YMMMb   
+
+   an interactive mind mapping tool
+```
+
+## What is this cool thing?
+
+An interactive mind map! We are using Node.js, Express, Javascript, JQuery, Pug and HTML5 Canvas to make a board that you can add ideas to, then link your ideas to other ideas. Possibly with a snazzy interface.
+
+We're testing with Mocha and Zombie and potentially Chai.
+
+## How do I get in on this?
+
+Not deployed to web yet- watch this space! We may also potentially deploy via Electron.
+
+## How do I use it?
+
+![mapp](https://github.com/charlottebrf/mytm/blob/lovely-readme/images/mindmapp.png)
+
+The green bar on the left has a selection of tools to help you map out your ideas- you can type a word in the text box, then click the shape that you would like to create. The idea will then appear on the canvas to the right, and can be clicked and dragged around with the mouse. You can then add lines between ideas by double clicking a start point and an end point.
+
+You can clear lines but leave ideas on the canvas by clicking the 'Clear Lines' button on the toolbar, or both lines and ideas by clicking 'Clear Canvas'. Different canvas colours can be toggled with the row of circular coloured buttons, and the toolbar can be shown or hidden with the toggle in the lower left.
+
+An instruction sheet can be toggled with the '?' button.
+
+## Our Approach
+
+We spent a good amount of time at the beginning of the project discussing user stories and the most appropriate technologies to use to accomplish them. We ultimately settled on using Node.js for the server and backend, and HTML5 Canvas for the mind map itself- meaning we would be using JavaScript to interface with it and render lines and shapes. The user interface was made with CSS3 and a very small amount of JavaScript for toggling parts of the display.
```

Comment:
Yep yep! I'll leave the latter JavaScript as is though because my UI JS isn't actually in ES6 sadly.